### ISSUE TITLE
[FEATURE] Setup Integration Test CI

### ISSUE THREAD

**Issue by jackiehanyang** [opensearch-project/opensearch-ai-flow-framework#56]
### Is your feature request related to a problem?
Currently, the integration test CI of this repo is not running. 

### What solution would you like?
Adding the proper Gradle setup to get the integration test CI running.

**Comment by dbwiddis** [comment]
In `build.gradle` they are configured:
```groovy
task integTest(type: RestIntegTestTask) {
    description = "Run tests against a cluster"
    testClassesDirs = sourceSets.test.output.classesDirs
    classpath = sourceSets.test.runtimeClasspath
}
tasks.named("check").configure { dependsOn(integTest) }

integTest {
    // The --debug-jvm command-line option makes the cluster debuggable; this makes the tests debuggable
    if (System.getProperty("test.debug") != null) {
        jvmArgs '-agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=*:5005'
    }
}
```

In CI action [it looks like it's running the task](https://github.com/opensearch-project/opensearch-ai-flow-framework/actions/runs/6301635983/job/17107138053):
> Task :integTest

**Comment by owaiskazi19** [comment]
@jackiehanyang I moved this issue under a wider integ test issue: https://github.com/opensearch-project/opensearch-ai-flow-framework/issues/73

**Comment by dbwiddis** [comment]
Closing this as it's addressed by #73
